File: csharp/Stats.cs
namespace lod
{
    public class Stats
    {
        // TODO add dexterity that will both help with soak and damage.
        //  but half of what strength gives.
        private int strength;

        public Stats(int strength)
        {
            this.strength = strength;
        }

        public int getStrength()
        {
            return strength;
        }
    }
}

File: go/stats.go
package codingdojo

type Stats struct {
	// TODO add dexterity that will both help with soak and damage.
	//  but half of what strength gives.
	strength int32
}

func makeStats(strength int32) Stats {
	return Stats{strength}
}

func (this Stats) getStrength() int32 {
	return this.strength
}
